astroid/brain/brain_argparse.py

```diff
@@ -9,8 +9,13 @@ def infer_namespace(node, context=None):
         # Cannot make sense of it.
         raise UseInferenceDefault()
 
-    class_node = nodes.ClassDef("Namespace", "docstring")
-    class_node.parent = node.parent
+    class_node = nodes.ClassDef("Namespace", parent=node.parent)
+    class_node.postinit(
+        bases=[],
+        body=[],
+        decorators=None,
+        doc_node=nodes.Const(value="docstring"),
+    )
```

Comment:
Is this necessary? From what I can tell, it can be removed. The tests also pass without it.